src/config.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+const fs     = require('fs');
+const moment = require('moment');
+const format = 'YYYY-MM-DD HH:mm:ss';
+const logger = require('./logger');
+
+const DEFAULTS = {
+  SNOW_PATH: 'api/fho/siam_in/create_transaction',
+  PROD_HOST: 'lssiprod.service-now.com',
+  TEST_HOST: 'lssitest.service-now.com'
+};
+
+const loadFromFile = file => { // eslint-disable-line consistent-return
+  if (file) {
+    try {
+      return fs.readFileSync(file, { encoding: 'utf-8' });
+    } catch (e) {
+      console.error('ERROR: Could not read from file:', file);
+      throw e;
+    }
+  }
+};
+const stripControl = text =>
+  text && text.replace(/\x1b\[\d+m/g, '').replace(/[\0-\x08\x11-\x1f]/g, ''); // eslint-disable-line no-control-regex
+
+const configure = env => { // eslint-disable-line complexity
```

Comment:
I don't really understand the benefit to making this a function. Surely it will only ever be run once and only with `process.env`?

Reply:
The point is to make testing easier (or possible even). The production run will only ever be `configure(process.env)`, but it allows much greater control for testing, and makes the tests themselves MUCH more straight forward.